src/bound/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod bound;
+
+pub use self::bound::*;
```

Comment:
db is basically nonexistant right now, but will be a bit more complex after we implement recovery from log on startup, pagecache LRU, and epoch gc.